data/ships/ships.json

```diff
@@ -0,0 +1,227 @@
+[
+  {
+    "name": "Modified YT-1300 Light Freighter",
+    "xws": "modifiedyt1300lightfreighter",
+    "ffg": "1"
```

Comment:
I'm not sure this file is really needed. I think we should add the `xws` and `ffg` fields to json files in `data/pilots/<faction>/<ship>.json`.

Reply:
The main reason I added this file is because we are starting to see more cross over ships such as the Tie/ln and Y-Wing. This way we can consolidate all of the ship specific features such as the dials, attack values etc removing the need to store them in the pilots "section" which could open to duplication of data and other potential issues.

Reply:
I'd prefer this info in the other ship json files for now. That way if someone wants to load all ship data they don't have to load this file and the others.

Reply:
Cool :) No issues with this at all and it sounds reasonable.